.github/workflows/downstream-native-image.yaml

```diff
@@ -7,15 +7,15 @@ on:
 # Keeping this file separate as the dependencies check would use more
 # repositories than needed this downstream check for GraalVM native image
 # compilation.
-name: downstream
+name: downstream-native-image
```

Comment:
The name was conflicting with another file (downstream.yaml). Maybe this is the reason why it wasn't showing up in pull requests correctly.